sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -1,23 +1,25 @@
 <template>
   <v-dialog v-model="showHideDialog" persistent :no-click-animation="true">
-    <v-card elevation="10">
+    <v-card elevation="10" :max-width="maxWidth">
       <v-card-header>
-        <v-card-title class="text-h5">
+        <v-card-title>
           <slot name="header">
-            <v-icon v-if="icon" class="mr-2" size="45">{{ icon }}</v-icon>
-            <span v-if="title" class="category-header-medium secondary-color">{{
-              title
-            }}</span>
+            <h2 v-if="title" class="category-header-large primary-color">
```

Comment:
w.r.t accessibility, just wanted to check, are we using `h2` for titles?

Reply:
I used this as a reference.
![image](https://user-images.githubusercontent.com/61259237/169140970-3e44ad08-6242-41e5-b248-3e8b65330717.png)